src/cum_sum_dist/dist_main.py

```diff
@@ -440,7 +442,9 @@ def dist_workflow(cfg):
             n_bootstraps = bootstrap_number
             print(f"Number of bootstraps in the trial run {n_bootstraps}")
             # Prepare arguments for each iteration
-            args_list = [(qmetric_residuals, dmask) for _ in range(n_bootstraps)]
+            args_list = [
+                (qmetric_residuals, dmask, seed_for_random) for _ in range(n_bootstraps)
+            ]
```

Comment:
You can use "multiplication" to make a list of repeated values:

```suggestion
            args_list = [(qmetric_residuals, dmask, seed_for_random)] * n_bootstraps            
```

Reply:
Thanks, @chuckwondo !